src/components/TodosList.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable react/prop-types, react/destructuring-assignment */
+import React from 'react';
+import TodoItem from './TodoItem';
+
+const TodosList = (props) => (
+  <ul>
+    {props.todos.map((todo) => (
+      <TodoItem
+        key={todo.id}
+        todo={todo}
+        handleChangeProps={props.handleChangeProps}
+        deleteTodoProps={props.deleteTodoProps}
+        setUpdate={props.setUpdate}
+      />
+    ))}
+  </ul>
+);
+export default TodosList;
```

Comment:
As discussed earlier, it would be better to implement prop de-structuring and add prop validations. You can checkout [this](https://www.npmjs.com/package/prop-types) documentation for the prop-types npm package.